### PR TITLE
magmatic_scorchwing.txt phyresis_roach.txt

### PR DESCRIPTION
**Pull request by Northmoc**
Two straightforward ones I did before I got sucked into working on SeekEffect 😁

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/magmatic_scorchwing.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Magmatic Scorchwing
+ManaCost:3 R R
+Types:Creature Dragon
+PT:4/4
+K:Flying
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | IsPresent$ Land.YouCtrl+Basic | PresentCompare$ EQ0 | PresentZone$ Library | Execute$ TrigDamage | TriggerDescription$ When CARDNAME enters the battlefield, if there are no nonbasic land cards in your library, CARDNAME deals 3 damage to any target.
+SVar:TrigDamage:DB$ DealDamage | ValidTgts$ Creature,Player,Planeswalker | TgtPrompt$ Select any target | NumDmg$ 3
+EQ0 | Execute$ TrigNoZombies | TriggerDescription$ At the beginning of your upkeep, if there are no Zombies on the battlefield, CARDNAME deals 1 damage to you.
```
Comment (Simisays) [inline forge-gui/res/cardsfolder/upcoming/magmatic_scorchwing.txt:8]: I assume this Execute is not needed ?

File: forge-gui/res/cardsfolder/upcoming/magmatic_scorchwing.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Name:Magmatic Scorchwing
+ManaCost:3 R R
+Types:Creature Dragon
+PT:4/4
+K:Flying
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | IsPresent$ Land.YouCtrl+Basic | PresentCompare$ EQ0 | PresentZone$ Library | Execute$ TrigDamage | TriggerDescription$ When CARDNAME enters the battlefield, if there are no nonbasic land cards in your library, CARDNAME deals 3 damage to any target.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/magmatic_scorchwing.txt:6]: nonbasic?

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/magmatic_scorchwing.txt:6]: Ha ha I totally read the oracle wrong 🤦

File: forge-gui/res/cardsfolder/upcoming/phyresis_roach.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Name:Phyresis Roach
+ManaCost:G
+Types:Creature Phyrexian Insect
+PT:1/1
+K:Toxic:1
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigEffect | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, Insects you control and Insect cards in your graveyard, hand, and library perpetually gain toxic 1.
+SVar:TrigEffect:DB$ Effect | Name$ Phyresis Roach's Perpetual Effect | RememberObjects$ Insect.YouCtrl,ValidGraveyard Insect.YouOwn,ValidHand Insect.YouOwn,ValidLibrary Insect.YouOwn | StaticAbilities$ PerpetualToxic | Duration$ Permanent | Triggers$ Update
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/phyresis_roach.txt:7]: that Remember can be write into a single "Valid..."

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/phyresis_roach.txt]: UGH copied the line from Lae'zel which had the only other bad use of course – well guess both will be fixed now!

**Review by tool4ever — APPROVED** [review]

**Pull request merged**